source/2019-11-01-the-ember-times-issue-122.md

```diff
@@ -9,19 +9,29 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Listen to an important announcement about Octane release 📝,
 Release of qunit-assertions-extra 🎉,
 Fine-tune validations with Ember Model Validator 3.10 🔍,
 READMORE
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Octane Release Update 📝](https://blog.emberjs.com/2019/10/31/octane-release-update.html)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Yesterday, the Ember team announced that [the upcoming 3.14 release will not include Octane as the default mode](https://blog.emberjs.com/2019/10/31/octane-release-update.html).
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+While [several developers are successfully using Octane in production already](https://twitter.com/mixonic/status/1180494661938552832), a few key ingredients are missing in order to make Octane **a success story for everyone**. The remaining tasks are:
+
+- Make removal of jQuery optional to help with your migration (completed in 3.14 ✅)
+- Finish updating [Ember Inspector](https://github.com/emberjs/ember-inspector) to support Glimmer components
+- Document the use of Glimmer components in the [Ember Guides](https://octane-guides-preview.emberjs.com/release/)
```

Comment:
What do you think about

```suggestion
- Update documentation with a renewed teaching model and about new Octane idioms, including the use of Glimmer components in the [Ember Guides](https://octane-guides-preview.emberjs.com/release/)
```

?

Reply:
@jessica-jordan Thanks for the suggestion!

I wanted to keep the sentence brief and getting-to-the-point, as we're handling public relations. I thought a longer summary might sound like we're defensive.

I read the Documentation section in the blog post again, and think we can combine our ideas as follows. What do you think about:  `Document the Octane programming model in the Ember Guides`? The term "programming model" would cover teaching both Glimmer components and Octane idioms.

Reply:
Yes, that totally makes sense to keep it brief and I think your suggestion sounds great as well!